pkg/sql/alisa_submitter.go

```diff
@@ -137,10 +120,12 @@ func (s *alisaSubmitter) cleanUpModel(modelPath string) error {
 	if len(sub) != 3 {
 		return fmt.Errorf("SQLFLOW_OSS_CHECKPOINT_DIR should be format: oss://bucket/?role_arn=xxx&host=xxx")
 	}
-	bucket, e := getBucket(sub[2], os.Getenv("SQLFLOW_OSS_AK"), os.Getenv("SQLFLOW_OSS_SK"), sub[1])
+	fmt.Println(os.Getenv("SQLFLOW_OSS_MODEL_ENDPOINT"), os.Getenv("SQLFLOW_OSS_AK"), os.Getenv("SQLFLOW_OSS_SK"), sub[1])
```

Comment:
Is this line printing a log message? Should we call `log.Print` or just remove it?

Reply:
Just used for debugging, I removed it.